components/esp_diagnostics/include/esp_diagnostics.h

```diff
@@ -126,13 +126,15 @@ typedef struct {
 /**
  * @brief Device information structure
  */
+ #define SHA_SIZE (16 / 2)
```

Comment:
Is there a need to redefine this macro? Can we pick a single place to define this?

Reply:
esp_dianostics.h and rtc_store.h  are independent hence it was needed. I guess other solution would be to have separate header file for such macros.